Source/WPF/MyMoney/Reports/NetWorthReport.cs

```diff
@@ -472,7 +475,7 @@ private void WriteRow(IReportWriter writer, Color color, string name, decimal ba
             writer.EndCell();
 
             writer.StartCell();
-            writer.WriteNumber(balance.ToString("C"));
+            writer.WriteNumber(this.myMoney.GetFormatedNormalizedAmount(balance));

```

Comment:
and there's a typo, double t, `GetFormattedNormalizedAmount`

Reply:
Formatting and Normalize are intentionally two concepts 
1) Formatting will simply do the localization text of the Amount
2) Normalize does the Ratio math conversion. 

That's why I wanted to be explicit in the naming of the functions

Reply:
fixed